octopoes/octopoes/models/ooi/config.py

```diff
@@ -0,0 +1,20 @@
+from typing import Literal
+
+from octopoes.models import OOI, Reference
+from octopoes.models.persistence import ReferenceField
+
+
+class Config(OOI):
+    object_type: Literal["Config"] = "Config"
+
+    ooi: Reference = ReferenceField(OOI)
+    bit_id: str
+    config: str
```

Comment:
Could be

Reply:
I think it would be more better to have the config specified as key value pairs and pass it to the normalizer as dict instead of having to parse json in every normalizer.